### ISSUE TITLE
Usability: Completely avoid using JS to navigate inside and outside of app

### ISSUE THREAD

**Issue by odzhychko** [nextcloud/calendar#8811]
### Is your feature request related to a problem? Please describe.

We currently use `window.open` to navigate the app.

### Describe the solution you'd like

Where possible it should be replaced with anchor elements `<a/>` for better usability
